app/views/layouts/angular-bootstrap/_provider_advanced_settings.html.haml

```diff
@@ -0,0 +1,27 @@
+%h3
+  {{ emsCommonModel.provider_options_description.advanced_settings.label }}
+%p
+  {{ emsCommonModel.provider_options_description.advanced_settings.help_text }}
+%div{"ng-repeat" => "settings_section in emsCommonModel.advanced_settings"}
+  %h4
+    {{ settings_section.label }}
+  %p
+    {{ settings_section.help_text }}
+  %div{"ng-repeat" => "option in settings_section.settings"}
+    %div{"ng-show" => "true"}
+      .form-group{"ng-attr-ng-class" => "{{ \"{'has_error': angularForm.provider_options_\" + settings_section.section_name + \"_\" + option.name + \".$invalid}\" }}"}
+        %label.col-md-2.control-label{"ng-attr-for" => "{{ \"provider_options_\" + settings_section.section_name + \"_\" + option.name }}"}
+          {{ option.label }}
+        .col-md-4
+          %input.form-control{"type"          => "text",
+                              "ng-attr-id"    => "{{ \"provider_options_\" + settings_section.section_name + \"_\" + option.name }}",
+                              "ng-attr-name"  => "{{ \"provider_options_\" + settings_section.section_name + \"_\" + option.name }}",
+                              "ng-model"      => "option.value",
+                              "checkchange"   => "",
+                              "ng-trim"       => false,
+                              "detect_spaces" => "",
+                              "placeholder"   => "{{ option.global_default }}",
+                              "title"         => "{{ option.help_text }}",
+                              "prefix"        => "provider_options"}
+          %span.help-block{"ng-attr-ng-show" => "{{ \"angularForm.provider_options_\" + settings_section.section_name + \"_\" + option.name + \".$error.detectedSpaces\" }}"}
```

Comment:
(We should really start putting these as sub-objects, and drop the idea of using prefixed fields, but .. I guess that's too big a change for the PR.)